### PR TITLE
refactor(components): [scrollbar] use useNamespace

### PR DESCRIPTION
**Pull request by buqiyuan**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/scrollbar/src/scrollbar.vue [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-  <div ref="scrollbar$" class="el-scrollbar">
+  <div ref="scrollbar$" :class="ns.b()">
     <div
       ref="wrap$"
       :class="[
         wrapClass,
-        'el-scrollbar__wrap',
-        native ? '' : 'el-scrollbar__wrap--hidden-default',
+        ns.e('wrap'),
+        native ? '' : ns.em('wrap', 'hidden-default'),
```
Comment (sxzz) [inline packages/components/scrollbar/src/scrollbar.vue:8]:
```suggestion
        { [ns.em('wrap', 'hidden-default')]: !native, }
```

**Review by sxzz — APPROVED** [review]

File: packages/components/scrollbar/src/scrollbar.vue (hunk 1 above)
Comment (buqiyuan) [reply packages/components/scrollbar/src/scrollbar.vue:8]: OK，Thank you for your suggestions

**Review by sxzz — APPROVED** [review]
LGTM. Thanks!
We will merge it after Feb 7.

**Comment by buqiyuan** [comment]
> LGTM. Thanks! We will merge it after Feb 7.

Okay, looking forward to the stable release later this year!

**Comment by sxzz** [comment]
Oh, please run `pnpm format`.

**Comment by buqiyuan** [comment]
> Oh, please run `pnpm format`.

ok i'm about to do this

**Pull request merged**